source/characterProcessing.py

```diff
@@ -519,6 +519,26 @@ def __init__(self, locale):
 			log.error("Invalid complex symbol regular expression in locale %s: %s" % (locale, e))
 			raise LookupError
 
+	def _replaceGroups(self, m, s):
+		res = ''

```

Comment:
Please add type hints and a doc string for this function. Generally we prefer more descriptive variable names.

Reply:
Ok, done so